lib/rules/font-weight-notation/index.js

```diff
@@ -36,7 +36,8 @@ const rule = function(expectation, options) {
       {
         actual: options,
         possible: {
-          ignore: ["relative"]
+          ignore: ["relative"],
+          ignoreFunctions: [_.isString]
```

Comment:
```suggestion
          ignoreFunctions: [_.isString, _.isRegExp]
```

To align with https://github.com/stylelint/stylelint/pull/3799